templates/projects.html

```diff
@@ -21,11 +21,11 @@
         <label for="check" class="checkbtn">
             <i class="fas fa-bars"></i>
         </label>
-        <label class="logo"><img src="../img/gdsc.png"
+        <label class="logo"><img src="https://lh3.googleusercontent.com/proxy/oTYrIbdN0Z2K0HMbxIv0ifcROdJOUz4etJX1e8-H_wwGe7Ru60yuP_oeyScyCsPl1AJPoCfy083p95JWKg"
                 alt=""></label>
         <ul>
             <li><a href="/">Home</a></li>
-            <li><a href="leaderboard.html">Leaderboard</a></li>
+            <li><a href="https://secret-inlet-87050.herokuapp.com/leaderboard">Leaderboard</a></li>
```

Comment:
Hardcoding URLs is really bad practice. Relative URLs should be used, since especially this URL (a non static Heroku URL) can change at any minute...